protoc-gen-frpc/templates/structs.templ

```diff
@@ -21,7 +21,8 @@
     {{end -}}
     type {{ CamelCase .FullName }} struct {
         error error
-        ignore bool
+        streamID uint32
```

Comment:
We should only generate the streamID field if we are doing streaming - otherwise we shouldn't generate it at all.

Reply:
Same with the status field since I can't imagine it being used in non-streaming examples